gradle/versions.gradle

```diff
@@ -162,5 +162,6 @@ dependencyManagement {
 
     dependency 'org.jupnp:org.jupnp.support:2.6.0'
     dependency 'org.jupnp:org.jupnp:2.6.0'
+    dependency 'io.jsonwebtoken:jjwt:0.9.1'
```

Comment:
This seems to be an old version - there are tags for 0.11.2.

Also, any particular reason to use this JWT library? I don't have a particular preference but it's slightly concerning it hasn't reached it's 1.0 release and there seem to be a number of potential alternatives that may be more mature. For example auth0 have a library (https://github.com/auth0/java-jwt)

Reply:
[mvn repository](https://mvnrepository.com/artifact/io.jsonwebtoken/jjwt) made me believe that it was the latest version. I used this library as it had the highest number of stars. But looking at it now it does have a lot more reported issues compared to `java-jwt`. I'll check out a few libraries and update